### PR TITLE
🦄 add prettierrc and format script on package.json

### PR DESCRIPTION
**Pull request by Mr-spShuvo**
## Code formatting with Prettier

This PR will enforce to follow code format as JS community standards using prettierrc.
A config file `prettierrc` has been added.
Also the a script has been introduced into the `package.json`.
Running `npm run format` or `yarn format` will format all the `js` file inside of the `src` directory.
Or you can use `prettier` plugin (works with most of the code editor) to automatic format your code.

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: frontend/src/Css/login.css [hunk 1]
```diff
@@ -3,133 +3,124 @@
...
-        text-align: center;
-        margin-bottom: 27px;
-        font-family: 'Ubuntu', sans-serif;
-        }
-
-
-.un:focus, .pass:focus {
-    border: 2px solid rgba(0, 0, 0, 0.18) !important;
-    
+.submit {
```
Comment (Vyvy-vi) [inline frontend/src/Css/login.css:104]: Was `.un` removed?
(Maybe the edits to the rest of the stuff could have been added in a second commit?)

**Review by Vyvy-vi — COMMENTED** [review]
Looks good to me!

File: frontend/src/Css/login.css (hunk 1 above)
Comment (Mr-spShuvo) [reply frontend/src/Css/login.css:104]: Nothing has been removed. Just prettier did it's job.

**Comment by i-am-zach** [comment]
Copy of #41 without Eslint

**Pull request closed without merging**